SoftLayer/CLI/vlan/list.py

```diff
@@ -16,7 +16,9 @@
            'datacenter',
            'hardware',
            'virtual_servers',
-           'public_ips']
+           'public_ips',
```

Comment:
In the table some titles start with a capital letter, but others are all lowercase, perhaps it should be ordered so that everything is with uppercase or lowercase